tests/FakeItEasy.Specs/CreationSpecs.cs

```diff
@@ -123,6 +123,24 @@ public interface ICollectionItem
                 .x(() => fakes.Should().OnlyContain(item => Fake.GetFakeManager(item) != null));
         }
 
+        [Scenario]
+        public void InterfaceWithAlikeGenericMethod()
+        {
+            "Given an interface with methods having multiple generic arguments that can be inheirited"
+                .See<IInterfaceWithSimilarMethods>();
+
+            "It should create fake instance of this"
+                .x(() => this.CreateFake<IInterfaceWithSimilarMethods>());

```

Comment:
We follow Given/When/Then, so the "It" is a little jarring. Perhaps

"When I create a fake of the interface"

and then store the created fake for interrogation in the next step, which can assert that "Then the fake is created" using `fake.Should().BeAFake()`